### PR TITLE
adding documentation feature

### PR DESCRIPTION
**Pull request by endersoncosta**
adding `usecase.doc()` feature to the usecase documentation

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "git.ignoreLimitWarning": true,
+    "workbench.colorCustomizations": {
+        "editorRuler.foreground": "#4e2d38",
+        "activityBar.background": "#173415",
```
Comment (italojs) [inline .vscode/settings.json:5]: we cant upload vscode customizatins, put it into .gitignore, please

**Review by italojs — CHANGES_REQUESTED** [review]

**Comment by italojs** [comment]
you PR has old commits from MAAAANNYYY people, update your branch with the herbs upstream master, please

**Comment by jhomarolo** [comment]
This pr is under conflict

**Comment by italojs** [comment]
this PR still alive? could we close it?

**Comment by dalssoft** [comment]
given the PR poor quality and how old it is, I'm closing this PR.

**Pull request closed without merging**